dynacred/src/personhood-rpc.ts

```diff
@@ -543,6 +643,44 @@ export class MeetupResponse extends Message<MeetupResponse> {
     }
 }
 
+export class Challenge extends Message<Challenge> {
+
+    static register() {
+        registerMessage("Challenge", Challenge);
+    }
+    readonly update: ChallengeCandidate;
+
+    constructor(props?: Properties<Challenge>) {
+        super(props);
+    }
+}
+
+export class ChallengeCandidate extends Message<ChallengeCandidate> {
+
+    static register() {
+        registerMessage("ChallengeCandidate", ChallengeCandidate);
+    }
+    readonly credential: InstanceID;
+    readonly score: number;
+    readonly signup: Long;
+
+    constructor(props?: Properties<ChallengeCandidate>) {
+        super(props);
+    }
+}
+
+export class ChallengeReply extends Message<ChallengeReply> {
+
+    static register() {
+        registerMessage("ChallengeReply", ChallengeReply);
```

Comment:
So, I've been using the variadic arguments to `registerMessage` to populate the message dependencies; as such, you need solely to register the "edge"-messages, the one visible and relevant to the outside.

```suggestion
        registerMessage("ChallengeReply", ChallengeReply, ChallengeCandidate);
```